frontend/src/app/scratch/[slug]/opengraph-image.tsx

```diff
@@ -29,6 +29,10 @@ export default async function ScratchOG({ params }: { params: { slug: string }})
     const score = compilation?.diff_output?.current_score ?? -1
     const maxScore = compilation?.diff_output?.max_score ?? -1
 
+    const percent = scratch.match_override ? 100 : calculateScorePercent(score, maxScore)
+    const doneWidth = Math.floor(percent * 300 / 100) // Unsure why this is 300 rather than 1200...
+    const todoWidth = 300 - doneWidth
```

Comment:
300 should be made into a const and used as the image width too 
And changed to 1200 with the other change I'm suggesting